bluepyemodel/model/model.py

```diff
@@ -250,12 +250,39 @@ def define_morphology(
         morph_modifiers_hoc = None
         do_replace_axon = True
     else:
+        morph_modifiers_hoc = [None] * len(morph_modifiers)
         for i, morph_modifier in enumerate(morph_modifiers):
             if isinstance(morph_modifier, list):
-                modifier_module = importlib.import_module(morph_modifier[0])
-                morph_modifiers[i] = getattr(modifier_module, morph_modifier[1])
+                try:
+                    modifier_module = importlib.import_module(morph_modifier[0])
+                    morph_modifiers[i] = getattr(modifier_module, morph_modifier[1])
+                    if len(morph_modifier) > 2:
+                        morph_modifiers_hoc[i] = morph_modifier[2]
+                    else:
+                        morph_modifiers_hoc[i] = getattr(modifiers, morph_modifier[1] + "_hoc")
```

Comment:
I don't think we should always try this _hoc trick when the user does not give a morph_modifiers_hoc. If it can't find it, we should give a warning, but continue the run.
Even if it is important to have the hoc for our pipeline, people might use this software outside of our pipeline, or just for part of it, and we should not force them to give a hoc string if they don't need it.
At least this is my opinion

Reply:
I agree, what do you think @darshanmandge ?

Reply:
I too agree with this